packages/terra-date-time-picker/src/terra-dev-site/common/DateTimePickerTimezoneExampleTemplate.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DateTimePicker from 'terra-date-time-picker/lib/DateTimePicker';
+import DateTimeUtils from '../../DateTimeUtils';
+
+const propTypes = {
+  /**
+   * The current entered date time. Use for the selected date message.
+   */
+  value: PropTypes.node,
```

Comment:
should this be a string value?

Reply:
updated here b5f07bc5f04b127f3d75bc46a3651a2e1091b165